README.md

```diff
@@ -79,14 +79,14 @@ The `sns-testing` solution is based on Docker; however, there are subtle issues
    ```
    and run the setup script:
    ```bash
-   bash-3.2$ ./setup_locally.sh
```

Comment:
The prefix `bash-3.2` is to emphasize that we run the script from within a bash shell on macOS. Are you confused by the version suffix `-3.2`?

Reply:
no i'm not, but you mention that fact at the beginning of the chapter and throughout the steps it's not really consistent afair, so i think it's better to get rid of the prefix for the sake of convenience (simple click to copy). also, i tested this in zsh and it works as well.

Reply:
Consistency is key, but I'd still like to make it explicit which commands must be executed from within Docker. 

How about we use something like

```bash
./setup_locally.sh  # inside the Docker container
```

throughout the instructions?

Reply:
Please review: https://github.com/letmejustputthishere/sns-testing/pull/1

Cc @letmejustputthishere @mraszyk

Reply:
The PR looks good to me, but shouldn't it also strip bash-3.2 from the bash command lines?

Reply:
Are you sure there are any remaining bash-3.2s in the latest README version? @mraszyk

Reply:
I wonder though whether the `# from Bash` comments still makes sense if we use Nix on Apple M1.

Reply:
We can always remove that later on, but it's not wrong.